modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -181,6 +182,43 @@ public void testAddObjectRelationship() throws Exception {
 		_objectRelationshipLocalService.deleteObjectRelationship(
 			objectRelationship);
 
+		ObjectField expectedObjectField = new ObjectFieldBuilder(
+		).externalReferenceCode(
+			RandomTestUtil.randomString()
+		).labelMap(
+			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString())
+		).readOnly(
+			ObjectFieldConstants.READ_ONLY_FALSE
+		).required(
+			RandomTestUtil.randomBoolean()
+		).build();
+
+		objectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				TestPropsValues.getUserId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				_objectDefinition2.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				"able", false, ObjectRelationshipConstants.TYPE_ONE_TO_MANY,
+				expectedObjectField);
+
+		ObjectField actualObjectField = _objectFieldLocalService.getObjectField(
+			objectRelationship.getObjectFieldId2());
+
+		Assert.assertEquals(
+			expectedObjectField.getExternalReferenceCode(),
+			actualObjectField.getExternalReferenceCode());
+		Assert.assertEquals(
+			expectedObjectField.getLabel(), actualObjectField.getLabel());
+		Assert.assertEquals(
+			expectedObjectField.getReadOnly(), actualObjectField.getReadOnly());
+		Assert.assertEquals(
+			expectedObjectField.isRequired(), actualObjectField.isRequired());
+
+		_objectRelationshipLocalService.deleteObjectRelationship(
+			objectRelationship);
```

Comment:
@devpedrotavares can we move this to the `_testAddObjectRelationshipOneToMany` method?